### ISSUE TITLE
Pyto Crashes During Sklearn Model Fittting

### ISSUE THREAD

**Issue by jammin0921** [ColdGrub1384/Pyto#63]
Congrats on porting scikit-learn to iOS! Incredibly impressive. 

Pyto crashes when I'm trying to fit a LogisticRegression model. LinearRegression seems to work fine but haven't cross-checked results yet to check accuracy.

**Comment by cclauss** [comment]
Steps to cause the crash?  Crash message / behavior?

**Comment by ColdGrub1384** [comment]
@cclauss I reproduced the bug with this code:

```python
from sklearn.datasets import load_iris
from sklearn.linear_model import LogisticRegression
X, y = load_iris(return_X_y=True)
clf = LogisticRegression(random_state=0, solver='lbfgs',
                         multi_class='multinomial').fit(X, y)
clf.predict(X[:2, :])
clf.predict_proba(X[:2, :]) 
clf.score(X, y)
```

That seems to be an issue with SciPy. The app crashes `EXC_BAD_ACCESS` at `f2py_rout__lbfgsb_setulb`, a function from SciPy. SciPy compilation method for iOS was very tricky, SciPy Fortran code was translated to C with [f2c](https://en.wikipedia.org/wiki/F2c) and then compiled instead of compiling Fortran directly. I think `f2py` is like an interface between Python and Fortran. The problem seems to be there.

**Comment by goodclass** [comment]
It's libfgsb's problem, but I tasted Analyser and it works. He may have found Fortran's compilation method. I contacted him before and asked him if he could share the technology. But he said that it took a lot of effort to accomplish it, and he hesitated a little. I'll contact him again these two days to see if I can get the compilation method by technology exchange or other means.

**Comment by goodclass** [comment]
https://github.com/goodclass/scipy_arm64/tree/master/scipy_build_2.0
Modified F2C header file
It's solved all the crashes I've encountered.
I continue to test other examples and data accuracy.

**Comment by ColdGrub1384** [comment]
Great! I will embed that.
